xbmc/messaging/ThreadMessage.h

```diff
@@ -59,10 +59,9 @@ class ThreadMessage
     , param1{ p1 }
     , param2{ p2 }
     , lpVoid{ payload }
-
+    , strParam{ param }
+    , params{ vecParams }
```

Comment:
Apparently, you're right. The Jenkins build for Win32 failed because of this reason. ThreadMessage.h(64): error C2797: 'KODI::MESSAGING::ThreadMessage::params': list initialization inside member initializer list or non-static data member initializer is not implemented (..\..\xbmc\addons\Skin.cpp) [c:\jenkins\slave\workspace\WIN-32\project\VS2010Express\XBMC.vcxproj]

*sigh*